### PR TITLE
ADAPT-2436 Add ref to CTA Button and CTA Link components

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Add ref to CTA Button and CTA Link components

# Needed By (Date)
- tbd

# Urgency
- low

# Steps to Test
1. Review code and confirm that the ref logic is structure the same as the [Button component](https://github.com/SU-SWS/decanter-react/tree/main/src/Button) 

# Associated Issues and/or People
- [ADAPT-2436](https://stanfordits.atlassian.net/browse/ADAPT-2436)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/CtaButton/CtaButton.stories.js [hunk 1]
```diff
@@ -168,6 +167,35 @@ CustomClick.args = {
...
-  onClick: () => { alert("Hey, you did it."); },
+  onClick: () => {
+    alert('Hey, you did it.');
+  },
 };
 CustomClick.storyName = 'Link with Custom Click';
+
+const ctaButtonRef = React.createRef();
+
+export const ForwardRef = ({ ...args }) => {
```
Comment (sherakama) [inline src/CtaButton/CtaButton.stories.js:178]:
```suggestion
export const ForwardRef = (args) => {
```

File: src/CtaLink/CtaLink.stories.js [hunk 2]
```diff
@@ -159,3 +159,29 @@ SrText.args = {
   href: 'https://decanter.stanford.edu',
 };
 SrText.storyName = 'Link with Screen Reader Only Text';
+
+const linkRef = React.createRef();
+
+export const ForwardRef = ({...args}) => {
```
Comment (sherakama) [inline src/CtaLink/CtaLink.stories.js:165]:
```suggestion
export const ForwardRef = (args) => {
```

**Review by sherakama — APPROVED** [review]
Two small suggested edits and then this is GTG.

**Comment by rebeccahongsf** [comment]
@yvonnetangsu Shea's suggestions have been merged. PR is ready for your review! 😄

**Comment by yvonnetangsu** [comment]
GTG - looks like Shea already pre-approved. Merging 🥳

**Pull request merged**